Backend/app/alchemist/spec/safety-dashboard/API/safetyDashboard.yaml

```diff
@@ -0,0 +1,40 @@
+imports:
+  Text: Data.Text
+  Users: Domain.Types.User
+  ShortId: Kernel.Types.Id
+  UTCTime: Kernel.Prelude
+
+
+
+module: Users
+types:
+  UsersResp:
+    dl : Text
+    firstName : Text
+    lastName : Text
+    flaggedStaus : FlaggedStatus
+    flaggedCounter : Int
+    updatedAt : UTCTime
+    createdAt : UTCTime
+
+
+apis:
+  # POST /upload/user
+  - POST:
+      endpoint: /upload/partner
+      params:
+
+      request:
+        type: API.Types.UI.Partner.PartnersReq
+      response:
+        type: API.Types.UI.Partner.PartnersResp
```

Comment:
Where are the files generated by these yamls?

Reply:
this is incomplete pr , work in progress